extensions/guacamole-auth-ldap/README

```diff
@@ -91,6 +91,11 @@ guacamole.properties such that the authentication provider is available.
 
     # The base DN within which all role based groups can be found.
     ldap-group-base-dn:      ou=groups,dc=example,dc=net
+    
+    # The additional search filter for the user to be authenticated. 
+    # for example , following filter indicate user have to be memeberof exampleGroup. otherwise, authentication will failed.
+    #the filter is also used to filter out all users visible to guacadmin
+    ldap-additional-search-filter: (memberOf=cn=exampleGroup,ou=groups,dc=example,dc=net) 
```

Comment:
Doesn't this also apply to the general user search, though?

I'd say `ldap-user-filter` sounds about right given its purpose.